### DISCUSSION TITLE
End-to-end encryption and security

### DISCUSSION THREAD

**Discussion by m3thm4th** [floccusaddon/floccus#1625] [Q&A]
Is Floccus E2EE? Meaning, if I use it with a NextCloud instance that doesn't belong to me, what will admins be able to see?

**Comment by marcelklehr** [comment]
Hi @m3thm4th 

If you sync via WebDAV you can enter an encryption password and your, but if you sync via Nextcloud Bookmarks your bookmarks are stored unencrypted on the server, because the Bookmarks app needs to be able to show them via the web UI.

**Comment by m3thm4th** [comment]
Thanks @marcelklehr

WebDAV passwords only protects the data in transit, as NextCloud does. But I don't understand if you implemented automatic data encryption for WebDAV, I found #224, but it's not clear to me. Does WebDAV automatically encrypt data? But the server admin can still see your WebDAV/NextCloud password with little effort, so they should be able to decrypt your data anyway. Would you mind explain?

**Reply by marcelklehr** [reply]
When setting up your webdav sync profile you need to enter your webdav authentication credentials. Optionally you can enter an additional passphrase which floccus will use to encrypt the file before uploading to webdav/nextcloud.
